### DISCUSSION TITLE
Why is this fork required?

### DISCUSSION THREAD

**Discussion by 0atman** [wrenger/obsidian-languagetool#28] [General]
Hi @wrenger, I assume this fork was needed because the last commit upstream was 10 months ago? That'd be fair enough.

If there's anything more official about the upstream plugin being abandoned, I couldn't find it quickly.
Thank you for your work!

**Comment by wrenger** [comment]
Hi @0atman, the primary reason for this fork was the number of changes I wanted to make. With the low frequency of updates to the original plugin, all the new features, fixes, refactoring changes would have taken a while to be merged. They might even be too big to be accepted, especially as the author stated that he stopped actively developing the plugin.

The first minor fixes actually were submitted and merged to the original.
